### PR TITLE
Defect-13091672

### PR DESCRIPTION
**Pull request by CHenrie**
@jbest84 Please Review

### CODE REVIEW

**Comment by slxbuilder** [comment]
Merged build triggered.

**Comment by slxbuilder** [comment]
Merged build started.

File: src/Views/OpportunityProduct/Edit.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ define('Mobile/SalesLogix/Views/OpportunityProduct/Edit', [
         discountText: 'discount',
         adjustedPriceText: 'adjusted price',
         quantityText: 'quantity',
-        extendedPriceText: 'extended price',
+        extendedPriceText: 'extended price',
```
Comment (jbest84) [inline src/Views/OpportunityProduct/Edit.js]: Unnecessary spaces.

File: src/Views/OpportunityProduct/Edit.js [hunk 2]
```diff
@@ -250,12 +250,13 @@ define('Mobile/SalesLogix/Views/OpportunityProduct/Edit', [
...
                     textProperty: 'Description',
-                    view: 'opportunity_related'
+                    view: 'opportunity_related',
+                    validator: validator.exists
                 },
                 {
                     name: 'ProductId',
                     property: 'ProductId',
-                    type: 'hidden'
+                    type: 'hidden'
```
Comment (jbest84) [inline src/Views/OpportunityProduct/Edit.js]: Unnecessary spaces.

**Comment by slxbuilder** [comment]
Merged build finished.

**Comment by jbest84** [comment]
@CHenrie Accept.

**Comment by CHenrie** [comment]
Retest

**Comment by slxbuilder** [comment]
Merged build triggered.

**Comment by slxbuilder** [comment]
Merged build started.

**Comment by slxbuilder** [comment]
Merged build finished.

**Comment by jbest84** [comment]
Validated as fixed by QA in DTS. Merging.

**Pull request merged**